Dockerfile

```diff
@@ -19,48 +19,24 @@ ARG ALPINE_VERSION=3.12
 FROM --platform=$BUILDPLATFORM alpine:${ALPINE_VERSION} AS git
 RUN apk add --no-cache git
 
-# xgo is a helper for golang cross-compilation
-FROM --platform=$BUILDPLATFORM tonistiigi/xx:golang@sha256:6f7d999551dd471b58f70716754290495690efa8421e0a1fcf18eb11d0c0a537 AS xgo
+# xx is a helper for cross-compilation
+FROM --platform=$BUILDPLATFORM tonistiigi/xx:golang@sha256:810dc54d5144f133a218e88e319184bf8b9ce01d37d46ddb37573e90decd9eef AS xx
+
+FROM --platform=$BUILDPLATFORM golang:1.13-alpine AS gostable
```

Comment:
(Can be another PR, but we seriously need to move away from Go 1.13)

Reply:
@tiborvass has a spreadsheet with all the possible `EINTR` for syscalls handlers for go1.13+ that needs to be confirmed to not have possible unfixed cases. @tiborvass please share with @AkihiroSuda 

But yes, unrelated to this PR.